sda-commons-starter-web/src/main/java/org/sdase/commons/spring/boot/web/security/handler/ValidationExceptionHandler.java

```diff
@@ -50,6 +56,13 @@ public ResponseEntity<ApiError> validationError(MethodArgumentNotValidException
     return ResponseEntity.unprocessableEntity().body(response);
   }
 
+  @ExceptionHandler({HandlerMethodValidationException.class})
+  public ResponseEntity<ApiError> validationErrorJakarta(HandlerMethodValidationException ex) {
+
+    LOG.error(ex.getMessage(), ex);
+    return ResponseEntity.unprocessableEntity().body(new ApiError(ERROR_MESSAGE));
+  }
```

Comment:
Can we give any hints to the consumers what went wrong? Now they will only see 422/"Validation error". Can we make use of the caught exception?

Reply:
added invalid params exclusively in case of field errors, have a look